spec/linter-tslint-spec.js

```diff
@@ -39,4 +39,20 @@ describe('The TSLint provider for Linter', () => {
       }),
     );
   });
+
+  it('handles null filepath', () => {
+    waitsForPromise(() =>
+      atom.workspace.open(null).then(editor => lint(editor)).then((result) => {
```

Comment:
`atom.workspace.open()` is sufficient here, and the second case is doing nothing since the problematic `textEditor.getPath()` doesn't rely on this direct input.